student_class.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person_class'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(id, age, classroom, name = 'UNKNOWN', parent_permission: true)
```

Comment:
- [ OPTIONAL ] Hey! 👋 In the Person class, I noticed that the id parameter is no longer being passed. Consequently, there's no need to inherit it in the superclass of the Student class. 🚫🆔 Let's streamline the code and remove any unnecessary parameters for clarity and efficiency. Thanks for optimizing! 👍

```suggestion
class Student < Person 
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
  def play_hooky
    "¯\(ツ)/¯"
  end
end
```